test/JDBC/input/forjson/forjsonauto-vu-prepare.sql

```diff
@@ -181,3 +181,15 @@ begin
 end;
 go
 
+CREATE PROCEDURE forjson_vu_p_6 AS
+BEGIN
+    select top 10
+        U.id,
+        U.firstname,
+        (select O.productId from forjson_auto_vu_t_orders O where O.userid = U.id
```

Comment:
scalar subquery can be also appear in where condition like : 

```
SELECT * FROM sq1 WHERE num = (SELECT MAX(num)/10 FROM sq2); 

```

Pls make sure added those test cases

Reply:
And also scalar subquery can be In GROUP BY clauses